cli/pack/systemd_dir.go

```diff
@@ -125,32 +122,64 @@ func getSystemdTemplate(projectCtx *project.ProjectCtx) (templates.Template, err
 		log.Warnf(
 			"App directory doesn't contain stateboard entrypoint script `%s`. "+
 				"Stateboard systemd service unit file wouldn't be delivered",
-			project.StateboardEntrypointName,
+			projectCtx.StateboardEntrypoint,
 		)
 	}
 
 	return &systemdFilesTemplate, nil
 }
 
+func getSystemdCtx(projectCtx *project.ProjectCtx) *systemdCtx {
+	var ctx systemdCtx
+
+	ctx.Name = projectCtx.Name
+	ctx.StateboardName = projectCtx.StateboardName
+
+	ctx.DefaultWorkDir = project.GetInstanceWorkDir(projectCtx, "default")
+	ctx.InstanceWorkDir = project.GetInstanceWorkDir(projectCtx, "%i")
+	ctx.StateboardWorkDir = project.GetStateboardWorkDir(projectCtx)
+
+	ctx.DefaultPidFile = project.GetInstancePidFile(projectCtx, "default")
+	ctx.InstancePidFile = project.GetInstancePidFile(projectCtx, "%i")
+	ctx.StateboardPidFile = project.GetStateboardPidFile(projectCtx)
+
+	ctx.DefaultConsoleSock = project.GetInstanceConsoleSock(projectCtx, "default")
+	ctx.InstanceConsoleSock = project.GetInstanceConsoleSock(projectCtx, "%i")
+	ctx.StateboardConsoleSock = project.GetStateboardConsoleSock(projectCtx)
+
+	ctx.ConfDir = projectCtx.ConfDir
+
+	ctx.AppEntrypointPath = project.GetAppEntrypointPath(projectCtx)
+	ctx.StateboardEntrypointPath = project.GetStateboardEntrypointPath(projectCtx)
+
+	if projectCtx.TarantoolIsEnterprise {
+		ctx.Tarantool = filepath.Join(projectCtx.AppDir, "tarantool")
+	} else {
+		ctx.Tarantool = "/usr/bin" // TODO
```

Comment:
Seems $PATH should be checked and $TARANTOOL_BIN/$TARANTOOL_DIR

Reply:
See https://github.com/tarantool/modulekit/blob/ckit/FindTarantool.cmake

Reply:
This is the PATH where Tarantool is placed on the target platform. It's `/usr/bin/tarantool` for centos, alt and ubuntu, but it can be set to current system Tarantool path. 

I agree that we need some reworks here, but I suggest to perform it later (e.g. when we start to use `runtime.txt`).